socialhome/streams/app/main.js

```diff
@@ -1,6 +1,6 @@
 import Vue from "vue"
-import {VueMasonryPlugin} from "vue-masonry"
 import BootstrapVue from "bootstrap-vue"
+import VueMasonryPlugin from "vue-masonry"
```

Comment:
This change must have been made a few days ago. With the version fixing in `package.json`, it's likely that the library was downgraded.